api/users/serializers.py

```diff
@@ -40,8 +73,11 @@ def absolute_url(self, obj):
         return obj.absolute_url
 
     def update(self, instance, validated_data):
-        # TODO
-        pass
+        assert isinstance(instance, User), 'instance must be a User'
+        for attr, value in validated_data.items():
```

Comment:
I am breaking out all the fields in social list into their own serializer fields, am I missing something that might cause trouble?